designer/src/test/java/org/restcomm/connect/rvd/TestUtils.java

```diff
@@ -72,10 +72,10 @@ public static File createDefaultProject(String projectName, String owner, File w
     }
 
     public static RvdConfiguration initRvdConfiguration(String contextToUse) {
-        URL url = TestUtils.class.getResource("contexts/" + contextToUse + "/restcomm-rvd.war");
+        URL url = TestUtils.class.getResource("/contexts/" + contextToUse + "/restcomm-rvd.war/WEB-INF/rvd.xml");
```

Comment:
still fails here. Don't waste any more time on this.

Reply:
works here sorry

Reply:
should we ignore test then?

Reply:
@jaimecasero . the only reason to ignore a test is if the test is wrong in principle. i.e. if its logic doesn't prove something. What we currently have is a test that runs differently on two environments. This is no reason to ignore it IMO.

However, I think there _is_ sth wrong with the principle/rational' of the test.  initRvdConfiguration() should be removed altogether and RvdConfiguration should be mocked using mockito as you beautifully did in RestcommClientTest. I took some time to fix that  :-).

Also, since you suggested this (ignoring the test) twice you should have a reason for that. Should the tests run clean from a higher perspective that i'm anaware of @jaimecasero ?